docs/source/Examples/lambci.rst

```diff
@@ -0,0 +1,312 @@
+	  --env-file $LAMBCI_PATH/env \
+	  -e DOCKER_LAMBDA_STAY_OPEN=1 \
+	  -p 9001:9001 \
+	  lambci/lambda:$LAMBCI_RUNTIME \
+	  $LAMBCI_HANDLER
+
+You may also want to create a script to streamline this process.
+See `Scripting LambCI Startup`_.
+
+*******************************************************************************
+Invoke the function
+*******************************************************************************
+
+Use SRK to invoke the ``echo`` function using the following command:
+
+::
+
+	./srk bench -b one-shot -a '{"hello" : "world"}'
+
+
+*******************************************************************************
+Shut down the LambCI server
+*******************************************************************************
+
+::
+
+	docker kill srk-lambci
+
+*******************************************************************************
+Using Custom Libraries
+*******************************************************************************
+
+SRK with LambCI allows you to add custom libraries to the runtime. The
+configuration procedure parallels that of AWS Lambda, which users
+`layers <https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html>`_.
+
+In this example we will create a runtime that includes Python's popular `Requests <https://requests.readthedocs.io/en/master/>`_.
+Use the commands below to install the layer:
+
+::
+
+	mkdir -p $HOME/lambci/layers/requests-layer/python
+	pip3 install requests -t $HOME/lambci/layers/requests-layer/python
+
+Now create a configuration file ``configs/lambci-local-requests.yaml`` as follows:
+
+::
+
+	default-provider : "lambci"
+
+	providers :
+	  lambci :
+	    faas : "lambciLambda"
+
+	service :
+	 faas :
+	    lambciLambda:
+	      # path to the lambci directory
+	      directory : "~/lambci"
+	      # address of lambci server API
+	      address : "localhost:9001"
+	      # runtime configuration
+	      runtimes :
+	        # with python requests package
+	        with-requests :
+	          # list of additional layers 
+	          layers :
+	            - "requests-layer"
+
+Now recreate the function using the requests layer:
+
+::
+
+	./srk --config configs/lambci-local-requests.yaml function create -s examples/requests -r with-requests
+
+
+Run the Docker command (see `Start the LambCI server`_).
+
+Invoke the function
+
+::
+
+	./srk --config configs/lambci-local-requests.yaml bench -b one-shot -a '{}'
+
+
+*******************************************************************************
+Using a custom runtime
+*******************************************************************************
+
+A custom runtime replaces the runtime environment provided by the FaaS provider
+with an own runtime package. This package has to be uploaded as a layer to the
+FaaS provider.
+
+To use a custom runtime, specify ``provided`` as the runtime name for the
+Docker command.
+
+::
+
+	$ ./lambci.sh ~/lambci provided lambda_function.lambda_handler
```

Comment:
The earlier examples don't assume you've scripted the launch. For this quick copy-paste example, it might be good to just put the docker command directly (we didn't require them to write the script).